explore/src/main/scala/explore/config/sequence/SequenceTableBuilder.scala

```diff
@@ -38,21 +43,22 @@ private trait GmosSequenceTableBuilder[S, D: Eq] extends SequenceRowBuilder[D]:
     ExtraRowColumnId -> FixedSize(0.toPx)
   ) ++ SequenceColumns.BaseColumnSizes
 
-  private def columns(using Logger[IO]): List[ColDef.Type] =
-    List(
-      SequenceColumns
-        .headerCell(HeaderColumnId, ColDef)
-        .withColumnSize(ColumnSizes(HeaderColumnId)),
-      ColDef(
-        ExtraRowColumnId,
-        header = "",
-        cell = _.row.original.value.toOption
-          .map(_.step)
-          .collect:
-            case step @ SequenceRow.Executed.ExecutedStep(_, _) =>
-              renderVisitExtraRow(step, showOngoingLabel = true)
-      ).withColumnSize(ColumnSizes(ExtraRowColumnId))
-    ) ++ SequenceColumns.gmosColumns(ColDef, _.step.some, _.index.some)
+  private lazy val columns: Reusable[List[ColDef.Type]] =
```

Comment:
The logger was passed down to `renderVisitExtraRow` but it was not used there. I removed it from there and here.